pr2eus/robot-interface.l

```diff
@@ -665,16 +666,19 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
               (send-all cacts :time-to-finish)))
            (t
             (send-all controller-actions :time-to-finish)))))
-  (:stop-motion (&key (stop-time 0))
-   "Stop current executing motion"
-   (let ((av (send self :state :potentio-vector)))
-     (send self :angle-vector av stop-time)
-     (send self :wait-interpolation)))
+  (:stop-motion (&key (stop-time 0) (wait t))
+   "Smoothly stops the motion being executed by sending the current joint state to the joint trajectory action goal"
```

Comment:
@Affonso-Gui how about  adding “, uses are expected to use :cancel-angle-vector, for most cases” or something.

Reply:
Are uses expected to use :cancel-angle-vector for most cases?

Reply:
if it is supported, what is the reason to use :stop-motion ?

Reply:
Both `:cancel-angle-vector` and `:stop-motion` are (should be) supported on all robots that use follow-joint-trajectory controllers.

> :cancel-angle-vector leads to an abrupt stop, while :stop-motion makes the robot stops smoothly. 

Although there are robots in which `:cancel-angle-vector` is smooth enough, prioritizing speed or smoothness is task/motion dependent, so I don't believe we can generalize into using one over the other.

Reply:
I see, then how about adding "note that :cancel-angle-vector leads to an abrupt stop, while :stop-motion makes the robot stops smoothly.", ":stop-time is transition time for stopping motion, but the actual interpolation duration is limited by :min-time of :angle-vector, which is set to 1.0 [sec] by default" etc, to avoid confusion like my case.

Reply:
@k-okada Added more information to the docstring. Please check to see if it's clear enough.